ts-frontend/src/components/Map.tsx

```diff
@@ -238,12 +347,86 @@ export const Map = () => {
             <GoogleMap
               mapContainerStyle={containerStyle}
               center={currentMapCenter}
-              zoom={12}
+              zoom={9}
             >
               <Marker
-                key="11111"
-                position={{ lat: adminLocation.lat, lng: adminLocation.lng }}
+                key={9999}
```

Comment:
this key should be the `currentUserUUID`

Reply:
done